docs/index.rst

```diff
@@ -46,32 +61,6 @@ parallelize, Just-In-Time compile to GPU/TPU, and more.
    glossary
    changelog
 
-.. toctree::
-   :maxdepth: 1
-   :caption: Advanced JAX Tutorials
-
-   notebooks/autodiff_cookbook
-   multi_process
-   notebooks/Parallelism_with_JAX
```

Comment:
It looks like this was removed here but not added in the other place. Maybe a sync issue, since this notebook was added relatively recently

Reply:
Ah so obvious. thanks for noting that